style.css

```diff
@@ -0,0 +1,70 @@
+body {
```

Comment:
- Kindly stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=VJYlSbMqq9yrCYDy-0) as much as possible.



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206639383-80da61f7-d0a4-42d2-845f-c82b75797bdf.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206640138-9035bf05-315c-492e-aaa2-b49edf5c7f96.png" width="200">|



😎️